README.md

```diff
@@ -1,6 +1,125 @@
+
+* Management IP is VLAN backed and configured on the host. vMotion & vSAN IP ranges are configured during the bring-up process.
+
+#### Hardware and ESXi Hosts
+
+* All servers are vSAN compliant and certified on the VMware Hardware Compatibility Guide, including but not limited to BIOS, HBA, SSD, HDD, etc.
+
+* Identical hardware (CPU, Memory, NICs, SSD/HDD, etc.) within the management cluster is highly recommended. Refer to vSAN documentation for minimal configuration.
+
+* One physical NIC is configured and connected to the vSphere Standard switch. The second physical NIC is not configured.
+
+* ESXi is freshly installed on each host. The ESXi version matches the build listed in the Cloud Foundation Bill of Materials.
+
+* All hosts are configured with a central time server (NTP). NTP service policy set to 'Start and stop with host'.
+
+* Each ESXi host is running a non-expired license - initial evaluation license is accepted. The bring-up process will configure the permanent license provided.
+
+#### Other Infrastructure
+
+* All hosts are configured with a DNS server for name resolution.
+
+## Custom root password
+
+### Generating custom root password
+
+To generate a password hash of your desired ESXi root password run the 'mkpasswd' command on a Linux system with the 'whois' package installed as follows
+<!-- TODO: there is no good way to generate this password on Windows or Mac. -->
+
+```shell
+mkpasswd --method=SHA-512
+```
+
+You'll be prompted to enter the desired password sting you wish to hash, then press enter.
+
+![Alt text](assets/9-mkpasswd_example.png "mkpasswd Example")
+
+The output will be the string you need to use in the `esxi_password` variable near the end of the `terraform.tfvars.example` file
+
+## Terraform Deployment Workflow
+
+### Preparation
+
+* Download the vcf-ems-deployment-parameter_X.X.X spreadsheet for the VCF version you're deploying from VMware and fill it out for your environment. For more about the "Cloud Builder Deployment Parameter Guide" spreadsheet file and its configuration, see [About the Deployment Parameter Workbook on docs.vmware.com](https://docs.vmware.com/en/VMware-Cloud-Foundation/5.2/vcf-deploy/GUID-08E5E911-7B4B-4E1C-AE9B-68C90124D1B9.html) (requires a login and entitlements).
+* Copy the terraform.tfvars.example file to terraform.tfvars and fill it with the same values you used in the vcf-ems-deployment-parameter_X.X.X spreadsheet.
+
+### Deployment
+
+* Deploy this Terraform module by running `terraform init -upgrade` and `terraform apply`.
+* RDP to the management host
+  * Username: `SYSTEM\Admin`
+  * Password: provided in the terraform output `terraform output -raw management_password`.
+* Download the Cloudbuilder OVA from VMware
+* Log in to one of the ESXi hosts
+  * Run `terraform output -raw esx01_address` and go to the address in a browser
+  * Our example uses: <https://sfo01-m01-esx01.sfo.rainpole.io>
```

Comment:
We should clarify that, at this step, you log in with the username `root` and the password whose hash was stored in `esxi_password`